g3/cli/cli.py

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
Shall we move this under `g3/generate/preview` package since it's going to be used for the preview of the generated messages?

Reply:
Yes of course, there is absolutely no strong opinion on this. Just placed them and named them to get going